src/components/Modals/RulesModal.vue

```diff
@@ -245,22 +250,49 @@
-                    <li>At the beginning of all of the player's turns, the game will choose either the "True" or "False" playing field as the "Active" part of each program to run.
+                    <li>At the beginning of all of the player's turns, the game will choose either the "True" or "False" playing field as the "Active" part of each program to run.</li>
+                    <li>You get bonus points for good programming practice. For more information on this go to the 'Bonus Points' tab.</li>
                     <li>The game is over if a player reaches the score limit in either paths:</li>
                     <ul>
-                      <li>Short: 15 instructions</li>
-                      <li>Medium: 25 instructions</li>
-                      <li>Long: 35 instructions</li>
+                      <li>Short: 75 instructions</li>
+                      <li>Medium: 100 instructions</li>
+                      <li>Long: 125 instructions</li>
                       </ul>
                   </ol>
                 </div>
+
+                <div role="tabpanel" class="tab-pane" id="bonusPoints">
+                  <h4>Bonus Points</h4>
+                  <p>Various cards, good coding practices, or game play types will give you bonus points!  </p>
+                  <ul id="bonusPointsList">
+                    <li><h3>Master Coder - 10 points per opponent</h3></li>
+                    <p>By completing your program before an opponent has played an instruction you will gain the Master Coder bonus of 10 points per opponent.</p>
+                    <li><h3>Defensive Programmer - 15 points</h3></li>
+                    <p>Defensive Programmer is a bonus for playing all 3 safety cards</p>
+                    <li><h3>Clean System - 10 points</h3></li>
+                    <p>Clean System is a bonus for not getting any viruses by the end of the game.</p>
+                    <li><h3>Cool System - 10 points</h3></li>
+                    <p>If you don't play any overclocking cards then you will be given a bonus 10 points.</p>
+                    <li><h3>Complete Program - 10 points</h3></li>
+                    <p>If you have a complete program on either path you'll get a bonus 10 points to be rewarded for your success.</p>
+                    <li><h3>Programming Practices Bonuses</h3></li>
+                    <p>You will be given several bonuses for using good programming practices: </p>
+                    <ol>
+                      <li><b>Grouping Cards - 5</b>: Grouping programs together is a good and essential programming practice, so you'll get a bonus score of 5 for using a group card.</li>
+                      <li><b>Repetition - 3</b>: Looping through programs is an efficient way of having instructions run multiple times, so you'll be given a bonus score of 3 for using a repetition card. </li>
```

Comment:
Change to: "Repetition in programs is ..."

I don't know if we have used the term "looping" before.